### DISCUSSION TITLE
query parameters on date helper

### DISCUSSION THREAD

**Discussion by kallepyorala** [mockoon/mockoon#746] [Help]
Is there a way to use query parameters in date helper? 

This doesn't work:
```
{{setVar 'dateFrom' (queryParam 'dateFrom')}}
{{setVar 'dateTo' (queryParam 'dateTo')}}
[
  {
    "date": "{{date dateFrom dateTo "yyyy-MM-dd'T'HH:mm:ss'"}}",
  }
]
```

This works but I can't format the date correctly:
```
{{setVar 'dateFrom' (queryParam 'dateFrom')}}
{{setVar 'dateTo' (queryParam 'dateTo')}}
[
  {
    "date": "{{faker 'date.between' dateFrom dateTo}}",
  }
]
```

**Comment by 255kb** [comment]
It should work but it's incompatible with the return value of `queryParam`. I will open an issue. 
I don't see an alternative right now. Also, Faker.js doesn't offer a formatting option. 
Thank you for reporting.

**Reply by 255kb** [reply]
Erratum: there is an alternative. You can use `queryParamRaw` instead of `queryParam` (which is kind of stringifying everything).
